### PR TITLE
Finished Writing Tests

### PR DESCRIPTION
**Pull request by KayleeLChan**
All of them pass!

### Change types

- [x] Non-breaking change
- [x] Adding tests
- [x] Front end change

### Testing
- [x] I have added testing to this PR

### CODE REVIEW

File: front-end/src/components/DragDropFile.jsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+        { dragActive && <div id="drag-file-element"
+        onDragEnter={handleDrag}
+        onDragLeave={handleDrag}
+        onDragOver={handleDrag}
+        onDrop={handleDrop}></div> }
+      </form>
+    );
+  };
+
+  export default DragDropFile;
```
Comment (WeissSa) [inline front-end/src/components/DragDropFile.jsx:71]: This file should not be in this PR. My recommendation to fix this quickly would just be to commit the deletion of the file, but save a personal copy elsewhere you can use for when you actually want to add it.

**Review by WeissSa — CHANGES_REQUESTED** [review]
Small change needed. Tests look good!

**Review by williamyao27 — APPROVED** [review]
Tests LGTM!

File: front-end/src/components/DragDropFile.jsx (hunk 1 above)
Comment (KayleeLChan) [reply front-end/src/components/DragDropFile.jsx:71]: oh yeah I forgot about it LOL thanks for the reminder

**Review by WeissSa — APPROVED** [review]
Girlbossed

**Review by helias22 — APPROVED** [review]

**Review by emilylwan — APPROVED** [review]
LGTM :)

**Pull request merged**